interfaces/Julia/ext/UnoMathOptInterfaceExt/MOI_wrapper.jl

```diff
@@ -834,7 +834,7 @@ function MOI.get(
     mult_g = Vector{Float64}(undef, model.inner.ncon)
     Uno.uno_get_constraint_dual_solution(model.solver, mult_g)
 
-    λ = mult_g[row(model, ci)]
+    # λ = mult_g[row(model, ci)]
```

Comment:
@cvanaret You still need \lambda for the function, why do you remove it?

Reply:
Wait a minute... it's used on line 852! What the hell.